api/payload.go

```diff
@@ -168,3 +168,16 @@ type Signal struct {
 type ErrorResponse struct {
 	Message string `json:"msg"`
 }
+
+// LogEntry is the payload for the StreamLog data.
+type LogEntry struct {
+	// Source is the source of the log entry. One of "shim" or "runtime".
+	Source string `json:"source"`
```

Comment:
Oh, I forgot to mention that the proxy still adds the client number to the log entry, just like today, except it's now another field of the "structured" log entry. That client number can still be used to differentiate between clients.